src/controller/CHIPDeviceControllerSystemState.h

```diff
@@ -182,6 +182,10 @@ class DeviceControllerSystemState
 #if CONFIG_NETWORK_LAYER_BLE
         mBleLayer = params.bleLayer;
 #endif
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+        mWiFiPafLayer = params.wifipaf_layer;
```

Comment:
That seems to be pre-existing, but to match other naming in the params shouldn't that member be called `wifiPAFLayer` or so, not `wifipaf_layer`?

And again: for `mWiFiPafLayer` we are being inconsistent in `Paf` vs `PAF`.  Please just use `PAF` throughout, or `paf` when it starts an identifier.